lib/lambda/sinkCpocs.ts

```diff
@@ -30,10 +24,7 @@ export const handler: Handler<KafkaEvent> = async (event) => {
   }
 };
 
-const officers = async (
-  kafkaRecords: KafkaRecord[],
-  topicPartition: string,
-) => {
+const officers = async (kafkaRecords: KafkaRecord[], topicPartition: string) => {
```

Comment:
Just curious, where does this const get its name from? Is "officers" a Kafka term, or does it have a OneMAC-specific meaning?

Reply:
These are irrelevant changes done by formatting. I've only changed the function call on line `73`